data/human/hails.txt

```diff
@@ -3112,17 +3112,23 @@ phrase "friendly militia 3"
 	word
 		"the Navy"
 		"the Republic"
+		"the Reps"
 		"Parliament"
+		"those representatives of Parliament"
 	word
 		" would "
-		" would come down here and "
-		" would stop hiding up in the Paradise Worlds and "
-		" would stop licking the boots of the Syndicate and "
+	word
+		""
+		"come down here and "
+		"stop hiding up in the Paradise Worlds and "
```

Comment:
I feel the "up" in this hail is unnecessary. What is it referring to? Even if whatever fleet that uses this hail is always "below" Paradise worlds, it's still an implication.

That said, "the" could also be taken out. Saying "the Paradise Worlds" is basically the same thing as saying "Paradise Worlds". Leaving:

".....stop hiding in paradise worlds and......"